frontend/tests/controllers/UserRegistrationTest.php

```diff
@@ -186,4 +186,33 @@ public function testUserParentalTokenNotGeneratedDueInvalidParameters() {
             $this->assertSame('parameterInvalid', $e->getMessage());
         }
     }
+
+    /**
+     * User 13 and its account link to the parent account
+     * on verification of parental token
+     */
+    public function testUse13LinkedToParentAccountWhenTokenVerificationDone() {
+        $defaultDate = strtotime('2022-01-01T00:00:00Z');
+        \OmegaUp\Time::setTimeForTesting($defaultDate);
+        // Create a 10 years-old user
+        ['identity' => $identity] = \OmegaUp\Test\Factories\User::createUser(
+            new \OmegaUp\Test\Factories\UserParams([
+                'birthDate' => strtotime('2012-01-01T00:00:00Z'),
+            ]),
+        );
+        $login = self::login($identity);
```

Comment:
The parent is the user who will verify the token, so you could create a new account for that new user, and login with this one:
```suggestion
        [
            'user' => $parentUser,
            'identity' => $parentIdentity,
        ] = \OmegaUp\Test\Factories\User::createUser();

        $login = self::login($parentIdentity);
```